docs/_layouts/custom.html

```diff
@@ -15,6 +15,57 @@
     $$\gdef \deriv #1 #2 {\frac{\D #1}{\D #2}}$$
     $$\gdef \pd #1 #2 {\frac{\partial #1}{\partial #2}}$$
     $$\gdef \set #1 {\left\lbrace #1 \right\rbrace} $$
+    
+    % My colours
+
+    $$\gdef \aqua #1 {\textcolor{8dd3c7}{#1}} $$
+    $$\gdef \yellow #1 {\textcolor{ffffb3}{#1}} $$
+    $$\gdef \lavender #1 {\textcolor{bebada}{#1}} $$
+    $$\gdef \red #1 {\textcolor{fb8072}{#1}} $$
+    $$\gdef \blue #1 {\textcolor{80b1d3}{#1}} $$
+    $$\gdef \orange #1 {\textcolor{fdb462}{#1}} $$
+    $$\gdef \green #1 {\textcolor{b3de69}{#1}} $$
+    $$\gdef \pink #1 {\textcolor{fccde5}{#1}} $$
+    $$\gdef \vgrey #1 {\textcolor{d9d9d9}{#1}} $$
+    $$\gdef \violet #1 {\textcolor{bc80bd}{#1}} $$
+    $$\gdef \unka #1 {\textcolor{ccebc5}{#1}} $$
+    $$\gdef \unkb #1 {\textcolor{ffed6f}{#1}} $$
+
+    % Vectors
+    $$\gdef \vx {\pink{\vect{x }}} $$
+    $$\gdef \vy {\blue{\vect{y }}} $$
+    $$\gdef \vb {\vect{b}} $$
+    $$\gdef \vz {\orange{\vect{z }}} $$
+    $$\gdef \vtheta {\orange{\vect{\theta }}} $$
```

Comment:
```suggestion
    $$\gdef \vtheta {\vect{\theta}} $$
```

I don't think we use theta as a latent in this year's write up.
It was used for the EBM lesson, but that was already transcribed last semester.